packages/Tag/src/Tag.styles.js

```diff
@@ -146,15 +90,150 @@ export const Delete = styled(Button.Close)(({ size }) => {
     height: ${width[size]};
     justify-content: center;
     min-height: auto;
-    padding: ${padding[size]};
     width: ${width[size]};
 
-    &:hover {
-      background: ${tokens.color.blackLighten60};
-    }
-
     &:focus {
       background: ${tokens.color.blackLighten60};
     }
   `;
 });
+
+const themeStyles = {
+  [types.themes.BLACK]: css`
+    background: ${tokens.color.black};
+
+    ${Delete} {
+      color: ${tokens.color.blackLighten70};
+      &:hover {
+        color: ${tokens.color.blackLighten60};
+      }
```

Comment:
It seems a lot of cases are either using this or, the `blackLigten30` and `blackLighten20` combo
For clarity, could you wrap the two cases into two templates, for example 
```
const lightBackground = css`
  ${Delete} {
      color: ${tokens.color.blackLighten70};
      &:hover {
        color: ${tokens.color.blackLighten60};
      }
    }
`;
```
and use it like this
```
  [types.themes.BLACK]: css`
    background: ${tokens.color.black};
    ${lightBackground}
  `;
```